app/src/main/java/com/example/cachua/musicdemo21/data/source/ContentResolverData.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.cachua.musicdemo21.data.source
+
+import android.content.ContentUris
+import android.content.Context
+import android.net.Uri
+import android.provider.MediaStore
+import com.example.cachua.musicdemo21.data.model.MusicModel
+
+class ContentResolverData {
+
+    private var mContext: Context
+    constructor(mContext: Context){
+        this.mContext=mContext
+    }
+
+    fun getData():ArrayList<MusicModel>{
+        var mArrayListMusic: ArrayList<MusicModel> = ArrayList()
+        val projection = arrayOf(MediaStore.Audio.Media._ID,
+                MediaStore.Audio.Media.ARTIST,
+                MediaStore.Audio.Media.TITLE,
+                MediaStore.Audio.Media.DATA,
+                MediaStore.Audio.Media.DISPLAY_NAME,
+                MediaStore.Audio.Media.DURATION,
+                MediaStore.Audio.Media.ALBUM_ID
+        )
+        val cr =  mContext.contentResolver
```

Comment:
```suggestion
        val contentResolver =  context.contentResolver
```
Nên đặt tên có nghĩa.